sponsor.html

```diff
@@ -394,11 +513,54 @@ <h2> Sponsorship Prospectus </h2>
     <section class="footer">
       Copyright &copy; 2019 - 2020 Technica. All Rights Reserved.
     </section>
-
   </article>
 </div>
 
 <!-- mlh badge -->
 <a id="mlh-trust-badge" style="display:block;max-width:100px;min-width:60px;position:fixed;right:50px;top:0;width:10%;z-index:10000" href="https://mlh.io/seasons/na-2020/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2020-season&utm_content=white" target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/2020/mlh-trust-badge-2020-white.svg" alt="Major League Hacking 2020 Hackathon Season" style="width:100%"></a>
+
+<script>
```

Comment:
Sorry, can we move this script to a separate file as well? That way the file is a bit shorter and more modularized

Reply:
yup ill move it to the same file as the plug in

Reply:
oh wait oops, did you mean the script that is initializing the tooltips?